packages/newspringchurchapp/src/prayer/PrayerTabView/PrayerTab.js

```diff
@@ -19,37 +19,42 @@ const StyledH4 = styled(({ theme }) => ({
   color: theme.colors.white,
 }))(H4);
 
-const PrayerTab = memo(({ prayers, type, title, description, ...props }) => (
-  <HorizontalPaddedView>
-    {prayers.length > 0 ? (
-      <View>
+const PrayerTab = memo(
+  ({ prayers, type, title, description, loading, ...props }) => (
+    <HorizontalPaddedView>
+      {prayers.length > 0 ? (
+        <View>
+          <VerticalPaddedView>
+            <StyledBodyText isLoading={loading} placeholder={'Loading Prayers'}>
+              {description}
+            </StyledBodyText>
+          </VerticalPaddedView>
+          <Button
+            title="Start Praying"
+            loading={loading}
```

Comment:
I don't think we should have a loading state here since we already have one above. if there's no data, it's gonna show a button that disappears, might look strange.

Reply:
are you leaving this one in there?